boards/Board.mk

```diff
@@ -37,6 +37,27 @@
 
 -include $(TOPDIR)/Make.defs
 
+ifneq ($(RCSRCS)$(RCRAWS),)
+ETCDIR := etctmp
+ETCSRC := $(ETCDIR:%=%.c)
+
+CSRCS += $(ETCSRC)
+
+RCOBJS = $(RCSRCS:%=%$(OBJEXT))
+
+$(RCOBJS): %$(OBJEXT): %
+	$(call PREPROCESS, $<, $@)
+	$(Q) mkdir -p $(dir $(ETCDIR)$(DELIM)$<)
+	$(Q) cp -r $@ $(ETCDIR)$(DELIM)$<
+
+$(ETCSRC): $(shell find $(RCRAWS) -type l) $(RCOBJS)
+	$(foreach raw,$(RCRAWS), \
+		$(shell cp --parents -rfLp $(raw) $(ETCDIR)))
```

Comment:
i suspect --parents is not portable.

usually -R is preferred over -r these days. do you need -r?

Reply:
Fixed.